src/applications/claims-status/tests/e2e/08.claim-letters.cypress.spec.js

```diff
@@ -8,153 +9,169 @@ const generateErrorContent = (title, code) => {
 };
 
 describe('Claim Letters Page', () => {
-  beforeEach(() => {
-    cy.intercept('GET', '/v0/claim_letters', claimLetters.data);
-    cy.intercept('GET', '/v0/feature_toggles?*', featureToggles);
+  context('feature toggle enabled', () => {
```

Comment:
The main change here is that I wrapped all of the existing tests in this `context` and created another at the bottom